jvm-libs/linea/clients/eth-logs-searcher/src/main/kotlin/linea/ethapi/Web3JLogsSearcher.kt

```diff
@@ -70,15 +66,109 @@ class Web3JLogsSearcher(
       }
   }
 
-  private fun findLogLoop(
+  override fun getLogsRollingForward(
```

Comment:
Not see its usage yet, but I would assume this would be the function used by message anchoring service?